packages/svelteui-core/src/components/Chip/Chip.styles.ts

```diff
@@ -0,0 +1,165 @@
+import { createStyles } from '$lib/styles';
+import type { DefaultProps, SvelteUIColor, SvelteUINumberSize, SvelteUISize } from '$lib/styles';
+import type { Component } from '$lib/internal';
+
+export interface ChipProps extends DefaultProps {
+	color?: SvelteUIColor;
+	id?: string;
+	checked?: boolean;
+	disabled?: boolean;
+	value?: string;
+	label?: Component | string;
+	radius?: SvelteUINumberSize | number;
+	size?: SvelteUISize;
+  type?: 'checkbox' | 'radio';
+  variant?: 'outline' | 'filled'
+	transitionDuration?: number;
+	wrapperProps?: { [key: string]: any };
+}
+
+export interface ChipStyleParams {
+	size: SvelteUISize;
+	color: SvelteUIColor;
+  variant: 'outline' | 'filled'
+	radius: SvelteUINumberSize | number;
+	transitionDuration: number;
+}
+
+export const sizes = {
+  xs: 24,
+  sm: 28,
+  md: 32,
+  lg: 36,
+  xl: 40,
+};
+
+const iconSizes = {
+  xs: 10,
+  sm: 12,
+  md: 14,
+  lg: 16,
+  xl: 18,
+};
+
+const padding = {
+  xs: 16,
+  sm: 20,
+  md: 24,
+  lg: 28,
+  xl: 32,
+};
+
+const checkedPadding = {
+  xs: 7.5,
+  sm: 10,
+  md: 11.5,
+  lg: 13,
+  xl: 15,
+};
+
+export default createStyles(
+	(theme, { color, radius, size, transitionDuration, variant }: ChipStyleParams, getRef) => {
+		return {
+      root: {},
+
+      label: {
+        ref: getRef('label'),
+        boxSizing: 'border-box',
+        color: theme.colorScheme === 'dark' ? theme.colors.dark50 : theme.colors.black,
+        display: 'inline-block',
+        alignItems: 'center',
+        userSelect: 'none',
+        border: '1px solid transparent',
+        borderRadius: theme.fn.radius(radius),
+        height: theme.fn.size({ size, sizes }),
+        fontSize: theme.fn.size({ size, sizes: theme.fontSizes }),
+        lineHeight: `${theme.fn.size({ size, sizes }) - 2}px`,
+        paddingLeft: theme.fn.size({ size, sizes: padding }),
+        paddingRight: theme.fn.size({ size, sizes: padding }),
+        cursor: 'pointer',
+        whiteSpace: 'nowrap',
+        transition: `border-color ${transitionDuration}ms ease, background-color ${transitionDuration}ms ease`,
+        WebkitTapHighlightColor: 'transparent',
+
+        // variants. not sure if this is right. Should it use the theme.fn.variant() helper?
```

Comment:
yap, using the `theme.fn.variant` helper. There are some usage examples in [Alert.svelte](https://github.com/svelteuidev/svelteui/blob/0f2e528339929626e48dc2c2054a58d49c62f5df/packages/svelteui-core/src/components/Alert/Alert.styles.ts#L35), if should already apply the correct colors, but I'm not sure it is 100% correct.

(nvm the `[${theme.dark} &]` bit, that was the old way of doing `darkMode` - comment below)